code/_onclick/click.dm

```diff
@@ -300,6 +302,16 @@
  */
 /mob/proc/ranged_secondary_attack(atom/target, modifiers)
 
+/**
+ * Right click
+ */
+/mob/proc/RightClickOn(atom/target, params)
+	return target.RightClick(src)
+
+/atom/proc/RightClick(mob/user)
```

Comment:
This and `RightClickOn` need to be documented, properly. Not just "right click", but what is params? What does the return value do?

https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md